@here/harp-map-theme/resources/berlin_tilezen_night_reduced.json

```diff
@@ -1179,19 +1126,9 @@
                     "size": 14.4,
                     "opacity": 0.6,
                     "minZoomLevel": 17,
-                    "roughness": 1.95,
```

Comment:
Same as above.

Reply:
Also text technique don't have this properties